bin/cap_to_nifti.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+""" Map CAP values into nifti space using combined atlas."""
+
+import pandas as pd
+import numpy as np
+import nibabel as nb
+from nilearn import plotting
+
+# load in CAP data and select only the relevant columns (i.e., remove index column)
+cap_data = pd.read_csv("data/enhanced_nki/desc-cap_groupmap.tsv", sep="\t")
+
+# reset index to be first column which is numbers 1-1054 for cap data
+caps = cap_data.set_index('Unnamed: 0')
+caps.index.names = ['Index'] #rename to be called index
+
+# load combined atlas
+combined = nb.load("data/parcellations/SchaeferTian_combined_MNI152_2mm.nii.gz")
+
+# get data from combined atlas
+com_atlas = combined.get_fdata()
+
+# store unique numbers from com_atlas as com_parcels
+com_parcels = (np.unique(com_atlas))
+# remove first row as it is zero
+com_parcels = np.delete(com_parcels, (0), axis=0)
+# remove rows after 1054 as we don't have corresponding CAP data 
+com_parcels = np.delete(com_parcels, slice(1054,1080), axis =  0)

```

Comment:
Okay this is not right. i will need to check the template I created.

Reply:
I have fixed the bug wrt the atlas - can you update your branch and use the new file, and update your script accordingly?
This is how you update your branch:
```
git checkout cap-to-nifti
git remote add upstream https://github.com/htwangtw/nkicap.git
git fetch upstream
git merge upstream/main
```

Reply:
Great, thanks for fixing. I've updated my branch, used the new file and updated the script accordingly.